### PR TITLE
Add initial idea list for GSoC

### PR DESCRIPTION
**Pull request by kasparm**
### Purpose
Add idea list for GSoC

### Where to look
docs/source/dev/idea_list.md

### Time to review
15 min

### CODE REVIEW

File: source/dev/idea_list.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+- Write Apache Airflow workflows that periodically 
+    - Query external data sources for availability
+    - Download external data sources
+    - Validate and clean the downloaded resources
+    - Archive downloaded resources to a blob store
+- Workflows are instrumented with telemetry
+- Telemetry are aggregated in InfluxDB
+- Metrics are displayed in a Grafana dashboard
+- If a data source is down, we throw an alarm
```
Comment (jon-hagg) [inline source/dev/idea_list.md:82]: I think we need a newline after the last bullet, it's showing the next sentence as a continuation

File: source/dev/idea_list.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+**Requirements:**
+Docker, Python, Grafana, PAAS, Apache Airflow
+
+**Level:** 
+Medium to Hard
+
+**Mentor:**
+
+### Project 4. - Test system for end-to-end testing of the simulation framework 
+Use [existing test power systems](http://labs.ece.uw.edu/pstca/) that will be used in our simulation framework.
```
Comment (jon-hagg) [inline source/dev/idea_list.md:94]: Would emphasize that this is test _data_ we want to integrate

File: source/dev/idea_list.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+This project is to use statistical and ML models to fill in missing energy demand and renewable energy profile information in a way which mimics the real data as close as possible.
+In our current datasets of regional energy demand, we have used averaging of adjacent days to fill in our missing data, as described [here](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
+
+We propose the following steps for this project:
+
+
+- Understand the data set and the [current methodology](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
+- Literature review to understand prediction models for time series data
+- Collect data to build training/validation/testing sets to rank algorithms using various metrics
+- Setup platform to crossvalidate with the data in order to evaluate algorithm performance
```
Comment (jon-hagg) [inline source/dev/idea_list.md:125]: I'd change "setup" to "set up" (the former is usually a noun). Actually, I might reorganize these two points slightly. Would have "collect data to build training/testing sets" as one point, then "evaluate algorithms using cross validation or other metrics" as the next one. Not sure it's necessary to be a "platform"

Comment (ToddG) [reply source/dev/idea_list.md:125]: I think 'crossvalidate' should probably be two words: 

https://en.wikipedia.org/wiki/Cross-validation_(statistics)

File: source/dev/idea_list.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+**Requirements:**
+Python, Pandas, understanding of statistics and ML concepts
+
+**Level:** 
+Medium
+
+**Mentor:**
+
+### Project 6. - Visualization
+Visualization of the power systems output data is essential for users to inteprete the simulations results.
```
Comment (ToddG) [inline source/dev/idea_list.md:137]: s/inteprete/interpret/

https://www.merriam-webster.com/dictionary/interpret

File: source/dev/idea_list.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers student a stipend to work on open source code during the summer.
```
Comment (ToddG) [inline source/dev/idea_list.md:3]: s/offers student/offers each student/

File: source/dev/idea_list.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers student a stipend to work on open source code during the summer. 
+If you are interested in the program [here](https://summerofcode.withgoogle.com/) you can find more information.
+
+If you are a student and new to GSoC read the following [guide](https://google.github.io/gsocguides/student/).
```
Comment (ToddG) [inline source/dev/idea_list.md:6]: ... new to GSoc, please read the ...

File: source/dev/idea_list.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+If you are new to BES [here](https://science.breakthroughenergy.org/) more about our organization.
```
Comment (ToddG) [inline source/dev/idea_list.md:7]: ... new to BES, look [here]...

File: source/dev/idea_list.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+
+## Community and Communication
+We are active on Slack which is the best way to reach us.
```
Comment (ToddG) [inline source/dev/idea_list.md:11]: ... on Slack and this is the best way ...

File: source/dev/idea_list.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+but you have something else in mind that would enhance our open source project,
+let's discuss it the #ideas channel on Slack.
+For each project we provided various scales to adapt to your skill, ambition or individual situation.
+
+Also keep in mind when planning out your work that for each project we expect you to follow our [contribution guide](contribution_guide.md).
+### Project 1. - Simulation Data Download Manager
+To run energy system simulations we need input data such as electrical demand, hydro flow, solar irradiance, and wind speed time series data among others. See here for [more details](https://github.com/Breakthrough-Energy/PreREISE#2-gather-data-for-simulation). 
+Currently the data is collected by using the package when needed. There is still a lot of manual labor involved in collecting and cleaning the data. Besides that the data source or API’s may change.
+
+The goal of this project is to encapsulate the existing code for downloading external resources into a single library and CLI.
```
Comment (ToddG) [inline source/dev/idea_list.md:36]: First time we use the CLI acronym so we should define it...

... into a single library and Command Line Interface (CLI).

No need to define it in the doc in subsequent uses.

File: source/dev/idea_list.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+The result of this project will be a single CLI tool that BE can use to download external resources, either manually, or by integrating into an orchestration/workflow system such as Apache Airflow.
```
Comment (ToddG) [inline source/dev/idea_list.md:37]: Since we are using BES as our acronym above, we should be consistent.

s/BE/BES/

File: source/dev/idea_list.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+**Level:** Easy
+
+**Mentor:**
+
+### Project 2. - Simulation Data Validation Manager 
+To run energy system simulations we need input data such as electrical demand, hydro flow, solar irradiance, and wind speed time series data among others. See here for [more details](https://github.com/Breakthrough-Energy/PreREISE#2-gather-data-for-simulation). 
+Currently the data is collected by using the package when needed. There is still a lot of manual labor involved in collecting and cleaning the data. Besides that the data source or API’s may change.
+
+The goal of this project is to encapsulate the validation and cleaning of external data resources in a into a single library and CLI. 
+The result of this project will be a single CLI tool that BE can use to validate external resources, either manually, or by integrating into an orchestration/workflow system such as Apache Airflow.
```
Comment (ToddG) [inline source/dev/idea_list.md:53]: s/BE/BES/

File: source/dev/idea_list.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+
+- Write a command-line tool that verifies previously downloaded external the data resources
```
Comment (ToddG) [inline source/dev/idea_list.md:55]: s/downloaded external the data/downloaded external data/

File: source/dev/idea_list.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+
+### Project 3. - Simulation Data Ingestion Pipeline
+To run energy system simulations we need input data such as electrical demand, hydro flow, solar irradiance, and wind speed time series data among others. See [here](https://github.com/Breakthrough-Energy/PreREISE#2-gather-data-for-simulation) for more details . 
+Currently the data is collected by using the package when needed. There is still a lot of manual labor involved in collecting and cleaning the data. Besides that the data source or API’s may change.
+
+The goal of this project is to automate the download, validation, and archival of external resources using industry standard components such as Apache Airflow, InfluxDB, Grafana, Blob Stores, etc. 
+
+Note that this project will eventually incorporate the work of the Simulation Data Validation Manager and Simulation Data Download Manager projects. However, each of these projects can proceed in parallel.
+
+
```
Comment (ToddG) [inline source/dev/idea_list.md:73]: delete this newline

File: source/dev/idea_list.md [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+- Write Apache Airflow workflows that periodically 
+    - Query external data sources for availability
+    - Download external data sources
+    - Validate and clean the downloaded resources
+    - Archive downloaded resources to a blob store
+- Workflows are instrumented with telemetry
+- Telemetry are aggregated in InfluxDB
+- Metrics are displayed in a Grafana dashboard
+- If a data source is down, we throw an alarm
+The workflows should be well documented with a focus on code quality, and using best-of-breed current technology stacks (Apache Airflow, Kubernetes, Helm, Docker, Python3, Grafana, etc.)
```
Comment (ToddG) [inline source/dev/idea_list.md:83]: s/and using/and use/

File: source/dev/idea_list.md (hunk 2 above)
Comment (ToddG) [inline source/dev/idea_list.md:94]: Possible rewording for line 94

Extend simulation framework to leverage existing power systems test grids.

File: source/dev/idea_list.md [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+**Mentor:**
+
+### Project 4. - Test system for end-to-end testing of the simulation framework 
+Use [existing test power systems](http://labs.ece.uw.edu/pstca/) that will be used in our simulation framework. 
+Generate input data to feed to the simulation engine using data science techniques.
+Write integration tests and produce documentation to showcase our open source software.
+
+- Collect data (python scripting)
+- Generate time series for demand, hydro, solar and wind matching the test grid (python data analysis)
+- Integrate the test model in our [simulation framework](https://github.com/Breakthrough-Energy/PowerSimData) (python)
```
Comment (ToddG) [inline source/dev/idea_list.md:100]: This sentence (line 100) seems to conflict slightly with line 94. What are we extending, what are we integrating, etc. I'm a little fuzzy on this after reading this.

File: source/dev/idea_list.md [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+- Write end-to-end tests (in docker container)
```
Comment (ToddG) [inline source/dev/idea_list.md:101]: Is the '(in docker container)' part relevant?

File: source/dev/idea_list.md [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+- Write documentation for the community (sphinx)
+
+**Requirements:**
+Object Oriented Programming in Python, Data analysis/plotting in Python (pandas, numpy, matplotlib, etc.) and docker
+
+**Level:** 
+Easy
+
+**Mentor:**
+[Ben](https://github.com/rouille)
```
Comment (ToddG) [inline source/dev/idea_list.md:112]: Feel free to reference me by my github profile as well:

Todd https://github.com/toddg

File: source/dev/idea_list.md [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+
+### Project 5. - Time-Series Modeling of Missing Energy Industry Data
+A significant barrier to realistic modeling of the power grid is missing data from public data sources.
+This project is to use statistical and ML models to fill in missing energy demand and renewable energy profile information in a way which mimics the real data as close as possible.
+In our current datasets of regional energy demand, we have used averaging of adjacent days to fill in our missing data, as described [here](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
+
+We propose the following steps for this project:
+
+
```
Comment (ToddG) [inline source/dev/idea_list.md:121]: delete extra newline

File: source/dev/idea_list.md [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+- Understand the data set and the [current methodology](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
```
Comment (ToddG) [inline source/dev/idea_list.md:122]: s/data set/dataset/ -- since we are using 'dataset' elsewhere.

File: source/dev/idea_list.md [hunk 20]
```diff
@@ -0,0 +1,148 @@
...
+- Literature review to understand prediction models for time series data
+- Collect data to build training/validation/testing sets to rank algorithms using various metrics
+- Setup platform to crossvalidate with the data in order to evaluate algorithm performance
+- Integrate most suitable algorithm(s) in our code base and generate new time series for our power system model
```
Comment (ToddG) [inline source/dev/idea_list.md:126]: How will this interpolation fit into the data ingestion pipeline?

File: source/dev/idea_list.md [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+Python, Pandas, understanding of statistics and ML concepts
+
+**Level:** 
+Medium
+
+**Mentor:**
+
+### Project 6. - Visualization
+Visualization of the power systems output data is essential for users to inteprete the simulations results.
+Our current plotting functions are base on matplotlib and bokeh.
```
Comment (ToddG) [inline source/dev/idea_list.md:138]: Project #6 really needs to be fleshed out. What kinds of visualizations might they be involved in? What's the cool data they have access to? What kinds of decisions might be made from this data and visualization?

**Review by ToddG — COMMENTED** [review]
* Ran aspell on this and found a couple of spelling errors.
* Updated some English phrases
* Normalize BE to BES

Project #4 is worded a bit confusingly
Project #6 needs more detail

Mentors not listed for Projects 1,2,3,5,6

File: source/dev/idea_list.md (hunk 3 above)
Comment (kasparm) [reply source/dev/idea_list.md:125]: @rouille, same here.

File: source/dev/idea_list.md (hunk 2 above)
Comment (kasparm) [reply source/dev/idea_list.md:94]: @rouille can you take care of this?

File: source/dev/idea_list.md (hunk 21 above)
Comment (kasparm) [reply source/dev/idea_list.md:138]: @merrielle will take care of this

File: source/dev/idea_list.md (hunk 19 above)
Comment (kasparm) [reply source/dev/idea_list.md:122]: Will go with "data set" for all the occurrences.

File: source/dev/idea_list.md (hunk 10 above)
Comment (kasparm) [reply source/dev/idea_list.md:37]: Will replace it with "we".

File: source/dev/idea_list.md (hunk 2 above)
Comment (kasparm) [reply source/dev/idea_list.md:94]: @rouille

File: source/dev/idea_list.md (hunk 15 above)
Comment (kasparm) [reply source/dev/idea_list.md:100]: @rouille

File: source/dev/idea_list.md (hunk 16 above)
Comment (kasparm) [reply source/dev/idea_list.md:101]: @rouille

File: source/dev/idea_list.md (hunk 17 above)
Comment (kasparm) [reply source/dev/idea_list.md:112]: @ToddG which projects should I add you?
I assume you are the primary mentor for #4

File: source/dev/idea_list.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers each student a stipend to work on open source code during the summer. 
+If you are interested in the program [here](https://summerofcode.withgoogle.com/) you can find more information.
```
Comment (rouille) [inline source/dev/idea_list.md:4]: here you can find more information --> you can find here more information

File: source/dev/idea_list.md [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+
+## Community and Communication
+We are active on Slack and this is the best way to reach us.
+[Join us here](http://science.breakthroughenergy.org/#get-updates), introduce yourself, discuss your potential project in the #ideas channel, and let us know if you have any questions.
+
+If you need to reach us via email, here you go: sciences@breakthroughenergy.org
+
+Please review our [code of conduct](code_of_conduct.md) before joining the community.
+## Application
+If you are a student and applying for GSoC find bellow the idea list and explore the projects we would like to accomplish with you this summer.
```
Comment (rouille) [inline source/dev/idea_list.md:18]: If you are a student and applying for GSoC find below the idea list --> If you are a student and applying for GSoC you will find below a list of projects we would like to accomplish with you this summer

File: source/dev/idea_list.md [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+
+When applying please provide:
+
+- A project proposal in markdown following the [Elements of a Quality Proposal guide](https://google.github.io/gsocguides/student/writing-a-proposal#elements-of-a-quality-proposal)
```
Comment (rouille) [inline source/dev/idea_list.md:22]: in markdown --> in a markdown file

File: source/dev/idea_list.md [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+- Motivation why you choose this particular project?
```
Comment (rouille) [inline source/dev/idea_list.md:23]: Explain your motivation for picking up the project

File: source/dev/idea_list.md [hunk 26]
```diff
@@ -0,0 +1,147 @@
...
+- Your resume with relevant experience in PDF format
```
Comment (rouille) [inline source/dev/idea_list.md:24]: Provide your resume with relevant experience in PDF format

File: source/dev/idea_list.md [hunk 27]
```diff
@@ -0,0 +1,147 @@
...
+## Idea list
+If you can’t find your ideal project on this list,
+but you have something else in mind that would enhance our open source project,
```
Comment (rouille) [inline source/dev/idea_list.md:27]: enhance --> benefit

File: source/dev/idea_list.md [hunk 28]
```diff
@@ -0,0 +1,147 @@
...
+let's discuss it the #ideas channel on Slack.
```
Comment (rouille) [inline source/dev/idea_list.md:28]: in the #ideas channel on Slack

File: source/dev/idea_list.md (hunk 24 above)
Comment (kasparm) [reply source/dev/idea_list.md:22]: That does not make sense with "When applying please provide:"

File: source/dev/idea_list.md [hunk 29]
```diff
@@ -0,0 +1,147 @@
...
+For each project we provided various scales to adapt to your skill, ambition or individual situation.
```
Comment (rouille) [inline source/dev/idea_list.md:29]: For each project, we provide prerequisites along with a  level of difficulty that should assist you in your decision.

**Comment by rouille** [comment]
I guess we are going to reword project 1 and 2 same don't use the exact same wording before and after the bullet list. Correct?

File: source/dev/idea_list.md (hunk 2 above)
Comment (rouille) [reply source/dev/idea_list.md:94]: To make it clear, I would rephrase the sentence as follows: 
Use [existing test power systems](http://labs.ece.uw.edu/pstca/) that will be used in our simulation framework a test data.

File: source/dev/idea_list.md (hunk 29 above)
Comment (kasparm) [reply source/dev/idea_list.md:29]: Would be nice if somebody would have time for it before the deadline.

File: source/dev/idea_list.md (hunk 20 above)
Comment (rouille) [reply source/dev/idea_list.md:125]: The ML algorithms will be used to improve the imputation of missing data. Those, along with the existing one in PreREISE, could be automatically called in Project 3

**Comment by kasparm** [comment]
> I guess we are going to reword project 1, 2 and 3 so we don't use the exact same wording before and after the bullet lists. Correct?

Would be nice if somebody would have time for it before the deadline.

File: source/dev/idea_list.md (hunk 15 above)
Comment (rouille) [reply source/dev/idea_list.md:100]: What about the following @ToddG:
Process and format the selected test grid model(s) so it can be integrated to the [PowerSimData](https://github.com/Breakthrough-Energy/PowerSimData) package (python)

File: source/dev/idea_list.md (hunk 16 above)
Comment (rouille) [reply source/dev/idea_list.md:101]: No it is not. We can link to [plug](https://github.com/Breakthrough-Energy/plug) so they can see what we are talking about

File: source/dev/idea_list.md (hunk 17 above)
Comment (rouille) [reply source/dev/idea_list.md:112]: @kasparm can we have several names under one project? If yes, you can add @ToddG here.

File: source/dev/idea_list.md [hunk 30]
```diff
@@ -0,0 +1,147 @@
...
+**Requirements:**
+Object Oriented Programming in Python, Data analysis/plotting in Python (pandas, numpy, matplotlib, etc.) and docker
+
+**Level:** 
+Easy
+
+**Mentor:**
+[Ben](https://github.com/rouille)
+
+### Project 5. - Time-Series Modeling of Missing Energy Industry Data
```
Comment (rouille) [inline source/dev/idea_list.md:114]: Time-Series Modeling of Missing Energy Industry  and Weather Data

File: source/dev/idea_list.md (hunk 24 above)
Comment (rouille) [reply source/dev/idea_list.md:22]: or you can drop the please provide

**Comment by rouille** [comment]
> > I guess we are going to reword project 1, 2 and 3 so we don't use the exact same wording before and after the bullet lists. Correct?
> 
> Would be nice if somebody would have time for it before the deadline.

I did something in d0d539f. @kasparm, do not hesitate to drop the commit if you don't like it

File: source/dev/idea_list.md (hunk 3 above)
Comment (rouille) [reply source/dev/idea_list.md]: I like what @jon-hagg  proposed

**Comment by kasparm** [comment]
> > > I guess we are going to reword project 1, 2 and 3 so we don't use the exact same wording before and after the bullet lists. Correct?
> > 
> > 
> > Would be nice if somebody would have time for it before the deadline.
> 
> I did something in [d0d539f](https://github.com/Breakthrough-Energy/docs/commit/d0d539fff8ff2c8efca2df73595357536afd3029). @kasparm, do not hesitate to drop the commit if you don't like it

Should we use Project 1a, 1b, 1c to show that they go together?

File: source/dev/idea_list.md (hunk 24 above)
Comment (kasparm) [reply source/dev/idea_list.md:23]: I went with the current version.

**Comment by kasparm** [comment]
We still need to assign mentors. Please indicate which project you would like to mentor, co-mentor.

**Comment by rouille** [comment]
> We still need to assign mentors. Please indicate which project you would like to mentor, co-mentor. 

I can help if necessary on 1a, 1b an 3 and mentor 2.

**Review by rouille — APPROVED** [review]
Nice

File: source/dev/idea_list.md [hunk 31]
```diff
@@ -0,0 +1,163 @@
...
+Visualization of complex systems like the power grid is essential for people to be more engaged and truly understand the impact of different policies. 
+The current visualizations are split between matplotlib and nivo, each being used by different teams.
+The goal of this project is to re-create these visualizations in a single, unified platform.
+
+- Convert [matplotlib](https://matplotlib.org/stable/index.html) graphs used by the research team to [plotly](https://plotly.com/python/)
+- Convert [nivo](https://nivo.rocks/) graphs used by the website team to plotly
+- Explore the data and generate interesting new graphs
+
+A participant with existing expertise might:
+- Create an extensible python script to automate the creation of plotly graphs that is triggered whenever new data is availible from the research team.
```
Comment (dmuldrew) [inline source/dev/idea_list.md:152]: Typo: Available

File: source/dev/idea_list.md [hunk 32]
```diff
@@ -0,0 +1,163 @@
...
+- Stand up a local endpoint to serve as a dashboard completed graphs
+- Generate more complex visualizations such as map graphs with availible mapbox tools
```
Comment (dmuldrew) [inline source/dev/idea_list.md:154]: Same: available

File: source/dev/idea_list.md [hunk 33]
```diff
@@ -0,0 +1,165 @@
...
+
+**Level:**
+Medium to Hard
+
+**Mentor:**
+[Todd](https://github.com/ToddG)
+[Jon](https://github.com/jon-hagg)
+
+### Project 2. - Test system for end-to-end testing of the simulation framework
+Use [existing power systems test grids](http://labs.ece.uw.edu/pstca/) in order to to benchmark our simulation framework
```
Comment (jon-hagg) [inline source/dev/idea_list.md:101]: Repeated "to"

File: source/dev/idea_list.md [hunk 34]
```diff
@@ -0,0 +1,165 @@
...
+## Idea list
+For each project, we provide prerequisites along with a level of difficulty that should assist you in your decision.
+If you can’t find your ideal project on this list
+but you have something else in mind that would benefit our open source project,
+let's discuss it in the #ideas channel on Slack.
+
+Also keep in mind when planning out your work that for each project we expect you to follow our [contribution guide](contribution_guide.md).
+
+### Project 1. - Simulation Data Manager
+To run energy system simulations we need input data such as electrical demand, hydro flow, solar irradiance, and wind speed time series data among others. You can read the  the [README of the PreREISE package](https://github.com/Breakthrough-Energy/PreREISE#2-gather-data-for-simulation) for more details.
```
Comment (jon-hagg) [inline source/dev/idea_list.md:36]: Repeated "the"

File: source/dev/idea_list.md (hunk 33 above)
Comment (jon-hagg) [reply source/dev/idea_list.md:101]: BTW found these in the logs from the [style check](https://github.com/Breakthrough-Energy/docs/pull/48/checks?check_run_id=1937181400).

Comment (kasparm) [reply source/dev/idea_list.md:101]: I should have looked there.

Comment (jon-hagg) [reply source/dev/idea_list.md:101]: I keep forgetting we had it

File: source/dev/idea_list.md [hunk 35]
```diff
@@ -0,0 +1,165 @@
...
+- Integrate most suitable algorithm(s) in our code base and generate new time series for our power system model
+
+**Requirements:**
+Python, Pandas, understanding of statistics and ML concepts
+
+**Level:**
+Medium
+
+**Mentor:**
+[Daniel Muldrew](https://github.com/dmuldrew)
```
Comment (rouille) [inline source/dev/idea_list.md:147]: Daniel Muldrew --> Daniel since we use first name only for other mentors

Comment (kasparm) [reply source/dev/idea_list.md:147]: I want to make sure students can find us on Slack and there are more than one Daniel.

**Comment by kasparm** [comment]
Going to rebase now.

File: source/dev/idea_list.md (hunk 20 above)
Comment (dmuldrew) [reply source/dev/idea_list.md:138]: This more that just interpolation, the idea is to leverage our existing data better to create a model to fill in the gaps. Interpolation is a more localized operation based on adjacent data points. There are some pretty huge gaps in the datasets which can’t be fixed with simple interpolation.

Comment (ToddG) [reply source/dev/idea_list.md:138]: @dmuldrew - is there a word that describes "fixing huge gaps in datasets" that we can use here?
